client/ayon_core/modules/loader_action.py

```diff
@@ -13,7 +13,6 @@ def tray_init(self):
         # Add library tool
         self._loader_imported = False
         try:
-            from ayon_core.tools.loader.ui import LoaderWindow
```

Comment:
Purpose of the method is to make sure the tool can be imported, removing the import makes it useless.